src/controllers/StorageController.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  JsonController,
+  Get,
+  Post,
+  QueryParam,
+  UploadedFile,
+  BadRequestError,
+  UseBefore,
+} from 'routing-controllers';
+import { AppUserService, AppUserServiceImpl, StorageService, StorageServiceImpl } from '@Services';
+import multer from 'multer';
+import { OfficerAuthMiddleware } from '@Middlewares';
+import { OpenAPI } from 'routing-controllers-openapi';
+
+const fileFilter = (req: Express.Request, file: Express.Multer.File, cb: Function) => {

```

Comment:
would be nice to have some documentation here linked to multer - initial reaction to this might just be why not promises

Reply:
done